### ISSUE TITLE
Job Submission new user notification email: set password link goes to wp-login.php page rather than Woocommerce

### ISSUE THREAD

**Issue by AlexxxH** [Automattic/WP-Job-Manager#1998]
**Describe the bug**
When a new user submits a job they are sent a new user registration email with a password reset link. The password reset link goes to the wp-login.php page. As WP Job Manager reccommends using Woocommerce for account management (https://wpjobmanager.com/document/registration-login-page/) I think this password reset should really go through Woocommerce instead, if Woocommerce is installed. Most Wordpress developers don't want users to see the wp-login.php page. 

**To Reproduce**
1. In Job Listings -> Settings -> Job submission, ensure "Enable account creation during submission", "Generate usernames from email addresses", and "Email new users a link to set a password" are all checked.
2. Whilst logged out, submit a job through WP Job Manager front-end job submission form
3. Receive registration details email with link to set a password
4. Click the link to set a password

**Screenshots**
![Screenshot from 2020-04-18 22-13-27](https://user-images.githubusercontent.com/48523389/79671588-37067f80-81c3-11ea-9452-8fc7838e85a2.png)
I've added a screenshot of the wp-login.php reset pasword page

**Expected behavior**
Reset password link should go through Woocommerce. For example, Woocommerce reset password functionality can be replicated as:
1. Logout
2. Go to woocomerce "accout" page
3. Click "Lost your password?" link
4. Enter email to send password reset email
5. Receive email and follow instructions to reset password

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WP Job Manager plugin is active.
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
- WordPress Version: 5.4
- WP Job Manager Version: 1.34.1
- PHP Version: 7.3.16
- Other important details: Woocommerce plugin activated

**Comment by thedigitalcircle** [comment]
Hi
Same thing observed on my system.

**Comment by tripflex** [comment]
WP Job Manager uses the standard `wp_new_user_notification` function to send the email, so if it's linking to the default `wp-login.php` page, that's because WooCommerce is not overriding that in any sense (basically this isn't related to WP Job Manager, but WordPress in general).

You can use my WP Login Flow plugin to customize the URL, login page, or anything else you may want:
https://wordpress.org/plugins/wp-login-flow/

Basically what's happening is the default WordPress email links specifically to the `wp-login.php` page, and unless WooCommerce adds specific handling to change this, it's always going to send the email with a link to `wp-login.php` page.

**Comment by alexsanford** [comment]
Note that you can also customize this email yourself using the `wp_new_user_notification_email` filter.

If we do this, it should probably at least be behind a setting or a filter. Given that there is already a filter that can be used, it seems unlikely that we would do this. But we can leave it open for further discussion 🙂
